docker/docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
       image: yangsuite:latest
       build:
         context: ./yangsuite
+        dockerfile: ./dockerfile
```

Comment:
The docker files have been renamed with uppercase D.  This is the wrong name.  Did you test this?

Reply:
yes michael, we have renamed the file so fixed to uppercase D

Reply:
Make sure you add this change to [this repository](https://wwwin-github.cisco.com/yang-suite/yangsuite) also.